backend/app.py

```diff
@@ -4,11 +4,20 @@
 from flask import Flask, request, abort, json, jsonify, render_template
 from werkzeug.exceptions import HTTPException
 
+
+
+
 from backend.ML.PolyReg import get_trend_pred
 from backend.ML.RNN import RNN
 
-template_dir = os.path.abspath('frontend/templates')
-static_dir = os.path.abspath('frontend/static')
+
+from bokeh.client import pull_session
+from bokeh.embed import server_session
+
+
+
+template_dir = os.path.abspath('Social-Good-Hackathon/frontend/templates')
```

Comment:
This should remain as before
template_dir = os.path.abspath('frontend/templates')